mods/ra/maps/monster-tank-madness/map.yaml

```diff
@@ -2101,6 +2101,7 @@ Rules:
 		-ConquestVictoryConditions:
 		MissionObjectives:
 			EarlyGameOver: true
+		EnemyWatcher:
 	World:
 		-CrateDrop:
```

Comment:
Yup, but this pr is specifically about fixing this mission, so the fix fits better here.